test/business-sync-business-config-spec.js

```diff
@@ -100,8 +100,8 @@ describe('business-sync business configuration document definition', function()
       doc,
       oldDoc,
       [
-        errorFormatter.supportedExtensionsAttachmentViolation('businessLogoAttachment', [ 'png', 'gif', 'jpg', 'jpeg' ]),
-        errorFormatter.supportedContentTypesAttachmentViolation('businessLogoAttachment', [ 'image/png', 'image/gif', 'image/jpeg' ]),
+        errorFormatter.supportedExtensionsAttachmentReferenceViolation('businessLogoAttachment', [ 'png', 'gif', 'jpg', 'jpeg' ]),
+        errorFormatter.supportedContentTypesAttachmentReferenceViolation('businessLogoAttachment', [ 'image/png', 'image/gif', 'image/jpeg' ]),
```

Comment:
These functions do not exist in the new validation-error-formatter module (they do still exist in the old, deprecated `etc/validation-error-message-formatter.js` though).